src/components/Loading/Loading.tsx

```diff
@@ -5,9 +5,6 @@ import {useAppBridge} from '../../utilities/app-bridge';
 
 export interface LoadingProps {}
 
-// This does have a display name, but the linting has a bug in it
-// https://github.com/yannickcr/eslint-plugin-react/issues/2324
-// eslint-disable-next-line react/display-name
```

Comment:
We've updated the version of eslint-plugin-react to a version that fixes this false positive